doc/source/changes.rst

```diff
@@ -6,6 +6,15 @@ Changes/Hints for migration
 This chapter will give you hints on how to handle version migration, in particular regarding what you may need
 to adapt in your project configuration, database etc. when upgrading to a new version.
 
+Version 2.5.6
+-------------
+Feature and maintenance release:
+
+* Improvements to the calculation of the number of pages of the table of contents
+* New parameter 'page_break_difference' allows customization of calculation of toc length
+* Initialization of pyramid_oereb processor at beginning of application start resulting in performance improvements.
+
```

Comment:
Should we mention here, that some (more) changes do only apply now if the instance is restarted? and what those elements are?

Reply:
I added a sentence ;)

Reply:
@svamaa thanks! 👍 
just a typo: conifguration  > configuration